recipes/meta-apo/meta.yaml

```diff
@@ -8,10 +9,11 @@ package:
 
 source:
   url: https://github.com/qibebt-bioinfo/meta-apo/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: ccd237ac02dc92decd0c309cedd0430af12d1a92b92f7c6b4207fdccc148fbb6
+  sha256: 77ba046e1d99c476f42fac2a85745f8d16bdfc5ed368499f7583717ed3e5b978
+
 
 build:
-  number: 0
+  number: 1
```

Comment:
You do not need to bump the build number as the version changed

Reply:
We have commit the version v1.1, so that if we don't change the build number, it will hint "ERROR: recipes/meta-apo/meta.yaml:16: build_number_needs_bump: The recipe build number should be incremented"

Reply:
Oh, so you first packaged v1.1, then switched to 1.0, and now you're updating back to 1.1, is that right? That's unusual, but why not

Reply:
Yes, that's exactly right!